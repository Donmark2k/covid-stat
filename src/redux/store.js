// import { configureStore } from '@reduxjs/toolkit';
// import { createLogger } from 'redux-logger';
// import rocketReducer from './rockets/rocketSlice';
// import missionReducer from './missions/missionSlice';

// const logger = createLogger({
//   // Options for the logger can be passed here
//   // Show only the actions in the console
//   predicate: (getState, action) => action.type !== 'SOME_ACTION',
//   // Collapse the logs by default
//   collapsed: true,
// });

// export const store = configureStore({
//   reducer: {
//     missions: missionReducer,
//     rockets: rocketReducer,
//   },
//   middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
// });

// export default store;
