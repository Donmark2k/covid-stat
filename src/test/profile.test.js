import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import { store } from '../redux/store';
import Profile from '../components/profile';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <React.StrictMode>
        <Provider store={store}>
          <BrowserRouter>
            <Profile />
          </BrowserRouter>
        </Provider>
      </React.StrictMode>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
