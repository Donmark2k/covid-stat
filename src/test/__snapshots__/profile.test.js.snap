// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="px-2 pr-2"
>
  <div
    className="row"
  >
    <div
      className="ps-lg-5 col-lg-6 col-md-6 col-sm-12"
    >
      <h2>
        My Missions
      </h2>
      <div
        className="list-group"
        onKeyDown={[Function]}
      >
        <div
          className="list-group-item"
          data-rr-ui-event-key={null}
          onClick={[Function]}
        >
          No missions have been reserved.
        </div>
      </div>
    </div>
    <div
      className="pe-lg-5 col-lg-6 col-md-6 col-sm-12"
    >
      <h2>
        My Rockets
      </h2>
      <div
        className="list-group"
        onKeyDown={[Function]}
      >
        <div
          className="list-group-item"
          data-rr-ui-event-key={null}
          onClick={[Function]}
        >
          No rockets have been reserved.
        </div>
      </div>
    </div>
  </div>
</div>
`;
